Modules/MFT/src/runDigitsRootFileReaderMFT.cxx

```diff
@@ -0,0 +1,144 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file    runDigitsRootFileReaderMFT.cxx
+/// \author  Guillermo Contreras
+///
+/// \brief This is an executable that reads digits from a root file from disk and sends the data to QC via DPL.
+///
+/// This is an executable that reads digits from a root file from disk and sends the data to QC via the Data Processing Layer.
+/// The idea is based in a similar reader by Andrea Ferrero and the DigitReaderSpec definition in MFT
+//  It can be used as a data source for QC development. For example, one can do:
+/// \code{.sh}
+/// o2-qc-run-digits-root-file-reader-mft --mft-digit-infile=some_data_file | o2-qc --config json://${QUALITYCONTROL_ROOT}/etc/your_config.json
+/// \endcode
+///
+
+// is this needed?
+/*
+#include <random>
+#include <iostream>
+#include <fstream>
+*/
+#include <vector>
+
+
+// root headers
+#include "TFile.h"
```

Comment:
Please use <> brackets to include anything that is outside of the QC project. With that, we reduce the compilation time a little bit by telling the compiler, that it should find these headers in external libraries.

Reply:
Done.